README.md

```diff
@@ -1,7 +1,8 @@
 ![UI5 logo](./docs/images/UI5_logo_wide.png)
 
 # [UI5](https://ui5.sap.com) Build and Development Tooling
-[![OpenUI5 Community Slack (#tooling channel)](https://slackui5invite.herokuapp.com/badge.svg)](https://slackui5invite.herokuapp.com)
+[![OpenUI5 Community Slack (#tooling channel)](https://img.shields.io/badge/slack-join-yellow.svg)](https://join-ui5-slack.herokuapp.com)
```

Comment:
Why yellow? 😄

Reply:
Couldn't find a green that matches the one of the travis badge :expressionless:

Reply:
What about this one? (Same color as the previous badge)
![https://img.shields.io/badge/slack-join-e01563.svg](https://img.shields.io/badge/slack-join-e01563.svg)
`https://img.shields.io/badge/slack-join-e01563.svg`

Reply:
Let's actually match the travis badge green 👍